landing-page/layouts/partials/header.html

```diff
@@ -39,6 +39,9 @@
                 <li>
                     <a class="page-scroll" href="{{ .Site.BaseURL }}community">Community</a>
                 </li>
+                <li>
+                    <a class="page-scroll" href="{{ .Site.BaseURL }}contribute">Contribute</a>
```

Comment:
I think it's worth adding a contribute tab, but how many tabs can we add in header? I hope we don't keep adding new tabs and cause issue in UI. @samredai any guideline here we should follow?

Reply:
I know some people have suggested drop-downs, which I think could work but I personally like the simplicity of just having the items there. I don't think we need to have top level links to every page. What if `/contribute` is only reachable by a link included in `community.md` and also a link in the iceberg repo's readme?

Reply:
have a link in community page also works for me.